### ISSUE TITLE
[Suggested Improve] Splitting the VueJS component

### ISSUE THREAD

**Issue by boomhq** [talvbansal/media-manager#58]
Hi, 

Thanks for your work it's awesome, juste maybe it's a good thing if your should split the VueJS component on standalone Git Repo, for publish an NPM package ?

**Comment by talvbansal** [comment]
Hi @boomhq,

Thanks for checking the project out and your suggestion!

Given the the media manager relies on the backend to work properly its not likely that I will split this out into a separate vue component. However with the project being open source you're more than welcome to fork it and remove the laravel / php code. If you need a hand in doing so I can assist once you've made some progress.

Is there a particular use case that you're trying to solve by splitting it out into something standalone?

Thanks

Talv

**Comment by talvbansal** [comment]
After going to a number of Javascript meetups i've been thinking more and more about this and think I am going to split this out into a separate package if possible.

**Comment by boomhq** [comment]
Hi @talvbansal 
Oops, I have not seen your previous message.
I agree it's a very good idea.
If I am I can make myself useful.

**Comment by talvbansal** [comment]
Excellent I'm going to start work on it over on this repo:
https://github.com/talvbansal/media-manager-js
Any help you can give would be much appreciated.

This should allow us to have a backend agnostic unit tested more organised application with this being the laravel integration for it.

**Comment by ruchern** [comment]
@talvbansal Pure VueJS frontend? I'm going to love it.

**Comment by talvbansal** [comment]
Yeah thats the plan! - I'd love any help you guys can give as i've not written a pure JS project before!

**Comment by ruchern** [comment]
@talvbansal I have forked the project, I will look into it over the weekend when I am free!
